### PR TITLE
Fix/instance in custom context

### PR DESCRIPTION
**Pull request by carmi2214**
A bug fix that enables passing an instance of some class in customContext,
for example inversify container

### CODE REVIEW

File: test/integration/server/context/test-context.ts [hunk 1]
```diff
@@ -1,10 +1,23 @@
 import { PolarisGraphQLContext, PolarisRequestHeaders } from '../../../../src';
 
+export class TestClassInContext {
```
Comment (osher-sade) [inline test/integration/server/context/test-context.ts]: Create a specific file for that class

File: test/integration/tests/custom-context.test.ts [hunk 2]
```diff
@@ -1,25 +1,50 @@
...
 import { graphQLRequest } from '../server/utils/graphql-client';
-import * as request from './jsonRequestsAndHeaders/customHeaders.json';
+import * as customContextCustomFieldRequest from './jsonRequestsAndHeaders/customContextCustomField.json';
+import * as customContextInstanceMethodRequest from './jsonRequestsAndHeaders/customContextInstanceMethod.json';
+import * as customHeadersRequest from './jsonRequestsAndHeaders/customHeaders.json';
 
 let polarisServer: PolarisServer;
 
-beforeEach(async () => {
+beforeAll(async () => {
```
Comment (osher-sade) [inline test/integration/tests/custom-context.test.ts]: why did you change it to `beforeAll`?
change it back to `beforeEach`

File: test/integration/tests/custom-context.test.ts [hunk 3]
```diff
@@ -1,25 +1,50 @@
...
     polarisServer = await startTestServer();
     await initializeDatabase();
 });
 
-afterEach(async () => {
+afterAll(async () => {
```
Comment (osher-sade) [inline test/integration/tests/custom-context.test.ts]: same here

**Review by osher-sade — APPROVED** [review]

File: test/integration/tests/custom-context.test.ts (hunk 2 above)
Comment (carmi2214) [reply test/integration/tests/custom-context.test.ts]: discussed on WhAtSaPp
